evently-firebase.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
```

Comment:
How can we not push these credentials, can we ignore this file or find a way of having this content in the environment variables.

Reply:
worked on this